prac_05/emails.py

```diff
@@ -0,0 +1,19 @@
+"""
+emails
+Estimate: 60 minutes
+Actual:    minutes
+"""
+
+store_email_name = {}
+email = str(input("Email: "))
+while email != "":
+    email_parts = [str(parts) for parts in email.split("@")]
+    whole_name = " ".join(email_parts[0].split(".")).title()
+    print(whole_name)
+    check_name = str(input(f"Is your name {whole_name}? (Y/n) "))
+    if check_name == 'n':
```

Comment:
Does this condition work if user enters 'no'? (Please have a look at sample output to see what happens when user enters 'no')